gis/routino/Portfile

```diff
@@ -22,16 +22,16 @@ license                 GPL-3
 master_sites            http://routino.org/download/
 extract.suffix          .tgz
 
-checksums               rmd160  ea9f8a1fd5f728f6f2a5382543be29ba1b2f1e00 \
-                        sha256  376512ef9627ce5c12e8c7a98dba7fe5125acfb89a9f9ee6758fbe36b5cf942f
+checksums               rmd160  d72b47a3742ca9647a2e983b3da00389ee3e536e \
+                        sha256  4b7174d76955e058986d343635e0a184385f2756fa4ffc02eb5e2399921e9db1
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
Done.